lib/catalog/martyrology.ts

```diff
@@ -3167,6 +3421,29 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Oliver Plunket',
       titles: [Title.Bishop, Title.Martyr],
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-omer-eveque
+    // - https://en.wikipedia.org/wiki/Audomar
+    omer_audomar_bishop: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Omer',
+      titles: [Title.Bishop],
+      dateOfBirth: 600,
```

Comment:
Neither of the sources state that the birth date is 100 % correct. Even the [French Wikipedia](https://fr.wikipedia.org/wiki/Audomar_de_Th%C3%A9rouanne#:~:text=aux%20environs%20de%20l%E2%80%99an) says _around the year 600_ _(aux environs de l’an 600)_.

```suggestion
      dateOfBirthIsApproximative: true,
      dateOfBirth: 600,
```